powerauth-webflow/pom.xml

```diff
@@ -32,6 +32,21 @@
     <description>Federated Authentication and Authorization Server for Web</description>
     <packaging>war</packaging>
 
+    <dependencyManagement>
```

Comment:
Is there any reason why not to put this dependency management into the parent pom?